src/ua/burkavtsov/hw11/ArrayList.java

```diff
@@ -0,0 +1,70 @@
+package ua.burkavtsov.hw11;
+
+import java.util.Objects;
+
+public class ArrayList<T> {
+    private Object[] array;
+    private int size;
+
+    public ArrayList() {
```

Comment:
We should have possibility to initialize size via constructor 
Please add constructor 
ArrayList(int size) {
        this.size = size;
        array = new Object[size];
    }

Reply:
Please, add a constructor